index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Odongo full-stack web-developer</title>
+</head>
+<body>
+    <link rel="stylesheet" href="style.css">
+    <header class="header">
+        <nav class="header-nav">
+            <p  class="MyLogo">
+                <a class="nav-brand" href="">My Logo</a>
+            </p>
```

Comment:
:orange_circle:  [OPTIONAL] I think you can use a div tag here in replacement of the p tag. The <p> tags are for paragraphs while the <div> tag defines a division or a section in an HTML document, and as you can see there is no paragraph here.